webapp/lib/logger.dart

```diff
@@ -31,6 +31,10 @@ trace(String f, String s) {
   }
 }
 
+severe(String s) {
+  _write(s);
```

Comment:
Could you add a TODO issue to implement a leaky bucket system for writing the logs? Otherwise a fast loop throwing exception could generate a lot of log traffic